docs/pages/guides/using-firebase.md

```diff
@@ -240,7 +244,9 @@ This sample was borrowed and edited from [this forum post](https://forums.expo.d
 
 In order to record analytics events, the Expo Firebase Core and Analytics packages needs to be installed.
 
-`expo install expo-firebase-analytics`
+<Terminal cmd={[  
+  '$ expo install expo-firebase-analytics'
+]} />
```

Comment:
Same as above. 😉

Reply:
Thanks a lot, I was not aware of this. Will keep in mind! I've changed this as per your suggestion.